vagrant/config/docker/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
what's the difference between this file and https://github.com/RackHD/on-build-config/blob/master/vagrant/config/mongo/config.json